contracts/abstracted_account.algo.ts

```diff
@@ -14,6 +14,8 @@ type PluginInfo = {
   cooldown: uint64;
   /** The last round the plugin was called */
   lastCalled: uint64;
+  /** Whether the plugin has permissions to change the admin account */
+  adminPrivileges: uint8;
```

Comment:
Is there a reason for using uint8 here rather than boolean?

Reply:
it refused to compile when i used a boolean :( seems like maybe theres a bug with using booleans inside box storage